app/models/menu-item.model.js

```diff
@@ -1,22 +1,20 @@
 module.exports = (sequelize, Sequelize) => {
-  const MenuItem = sequelize.define("menuitem", {
+  const MenuItem = sequelize.define('menuItem', {
     title: {
-      type: Sequelize.STRING
+      type: Sequelize.STRING,
     },
     description: {
-      type: Sequelize.STRING
+      type: Sequelize.STRING,
     },
     image: {
-      type: Sequelize.STRING
-    }
+      type: Sequelize.STRING,
+    },
   });
 
   MenuItem.associate = (models) => {
-    MenuItem.belongsToMany(models.MenuItem, {
-      through: 'MenuToMenuItems',
-      as: 'Menus',
```

Comment:
You don't need the "as" or "foreignKey" attributes - sequelize will fill them in for you with some sane defaults.